### DISCUSSION TITLE
New Editor

### DISCUSSION THREAD

**Discussion by R4356th** [Glitchy-Scratch/glitchy-scratch.github.io#24] [Announcements]
Hello everyone,

Glitchy Scratch finally has an editor now! You can access it at https://glitchy-scratch.js.org/gui/master/! We are working on making changes and adding bugs to it. You are strongly encouraged to make pull requests to add bugs to our forks of Scratch's repositories!

Thank you.

**Comment by TheSmallyDog** [comment]
I have a suggestion on a glitch, sometimes a vector sprite or a vector background will turn to a gray box with a question mark.
